contrib/IECoreUSD/src/IECoreUSD/USDScene.cpp

```diff
@@ -213,8 +216,38 @@ void writeSetInternal( const pxr::UsdPrim &prim, const pxr::TfToken &name, const
 	collection.CreateIncludesRel().SetTargets( targets );
 }
 
+boost::container::flat_map<IECore::InternedString, pxr::TfType> g_schemaTypeSets = {
+	{ "__cameras", pxr::TfType::Find<pxr::UsdGeomCamera>() },
```

Comment:
This `__cameras` convention isn't used in SceneCache yet (just internally at IE via Caribou). If we're adding it for USDScene, we might as well add it for SceneCache too.

Reply:
Similarly for lights if/when we allow loading lights via USD.

Reply:
I won't be able to justify the time on that right now, since all SceneCache use I'm aware of is at IE and already has `__cameras` sets authored. Feel free to schedule it on your side though (and for IECoreAlembic if you're feeling generous). I'm not overly enamoured of the `__cameras` convention and definitely willing to reconsider as part of a larger alignment of all things.

Reply:
Sure thing, we can do it when we get around to it. Totally happy to change the name too, `__cameras` is quite mysterious but users still manage to want to use it...

Reply:
Yeah, the original principal was that it was private to gaffer, but in practice SceneInterfaces need to provide it, and it's a bit useful as a user, and we've had to munge the name to show it in the UI as if it was public anyway.